hotshot-query-service/src/data_source/storage/sql/queries/state.rs

```diff
@@ -854,7 +855,8 @@ mod test {
                 .await
                 .unwrap();
             assert_eq!(proof.elem(), None);
-            assert!(test_tree.non_membership_verify(100, proof).unwrap());
+
+            assert!(UniversalMerkleTree::<usize, Sha3Digest, usize, 8, Sha3Node>::non_membership_verify(test_tree.commitment(), 100, proof).unwrap());
```

Comment:
@alxiong @imabdulbasit just curious, we don't have the previous API anymore? just looks a lot simpler.